<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaPrincipal</class>
 <widget class="QMainWindow" name="PantallaPrincipal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>607</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ASPIRADORA ROBOT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes_pantalla_principal.qrc">
    <normaloff>:/fotos/fotos_pantalla_principal/fondo_app.png</normaloff>:/fotos/fotos_pantalla_principal/fondo_app.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">


background-color:rgb(255, 234, 197);




/*background-color:rgb(171, 212, 255);*/</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titulo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
/*background-color:rgb(55, 95, 180);*/
background-color:rgb(239, 239, 239);
/*border-radius:50px;*/
border:2px solid black;
}
QPushButton:hover{
background-color:grey;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#000000;&quot;&gt;ASPIRADORA ROBOT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonConfig">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>14</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imagenes_pantalla_principal.qrc">
       <normaloff>:/fotos/fotos_pantalla_principal/ajuste_dos.png</normaloff>:/fotos/fotos_pantalla_principal/ajuste_dos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoRepeatDelay">
      <number>500</number>
     </property>
     <property name="autoRepeatInterval">
      <number>500</number>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="ImagenAspiradora">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>161</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="imagenes_pantalla_principal.qrc">:/fotos/fotos_pantalla_principal/aspiradora.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>220</y>
          <width>171</width>
          <height>121</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QPushButton" name="botonArriba">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>0</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:20px;
border:2px solid black;
background-color:rgb(239, 239, 239);
}
QPushButton:enabled
{
background-color:rgb(239, 239, 239);
}
QPushButton:disabled
{
background-color:rgb(255,105,97);
}
QPushButton:hover{
background-color:grey;
}
QPushButton:checked
{
border:2px solid green;
}
QPushButton:unchecked
{
border:2px solid black;
}
/*background-color:rgb(239, 239, 200);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagenes_pantalla_principal.qrc">
           <normaloff>:/fotos/fotos_pantalla_principal/arriba.png</normaloff>:/fotos/fotos_pantalla_principal/arriba.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="botonDerecha">
         <property name="geometry">
          <rect>
           <x>113</x>
           <y>57</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:20px;
border:2px solid black;
background-color:rgb(239, 239, 239);
}
QPushButton:enabled
{
background-color:rgb(239, 239, 239);
}
QPushButton:disabled
{
background-color:rgb(255,105,97);
}
QPushButton:hover{
background-color:grey;
}
QPushButton:checked
{
border:2px solid green;
}
QPushButton:unchecked
{
border:2px solid black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagenes_pantalla_principal.qrc">
           <normaloff>:/fotos/fotos_pantalla_principal/derecha.png</normaloff>:/fotos/fotos_pantalla_principal/derecha.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="botonIzquierda">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>57</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:20px;
border:2px solid black;
background-color:rgb(239, 239, 239);
}
QPushButton:enabled
{
background-color:rgb(239, 239, 239);
}
QPushButton:disabled
{
background-color:rgb(255,105,97);
}
QPushButton:hover{
background-color:grey;
}
QPushButton:checked
{
border:2px solid green;
}
QPushButton:unchecked
{
border:2px solid black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagenes_pantalla_principal.qrc">
           <normaloff>:/fotos/fotos_pantalla_principal/izq.png</normaloff>:/fotos/fotos_pantalla_principal/izq.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="botonAbajo">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>57</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:20px;
border:2px solid black;
background-color:rgb(239, 239, 239);
}
QPushButton:enabled
{
background-color:rgb(239, 239, 239);
}
QPushButton:disabled
{
background-color:rgb(255,105,97);
}
QPushButton:hover{
background-color:grey;
}
QPushButton:checked
{
border:2px solid green;
}
QPushButton:unchecked
{
border:2px solid black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagenes_pantalla_principal.qrc">
           <normaloff>:/fotos/fotos_pantalla_principal/abajo.png</normaloff>:/fotos/fotos_pantalla_principal/abajo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>343</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>345</width>
         <height>441</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
/*background-color:rgb(55, 95, 180);*/
background-color:rgb(239, 239, 239);
border-radius:50px;
border:2px solid black;
}
QPushButton{
/*font:87 15pt &quot;Arial Black&quot;;*/
font: 63 15pt &quot;Yu Gothic UI Semibold&quot;;
background-color:rgb(255, 234, 197);
color:rgb(0, 0, 0);
border-radius:10px;
border:2px solid black;
}
QPushButton:hover{
background-color:grey;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QPushButton" name="botonInicio">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Iniciar limpieza&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>INICIAR</string>
        </property>
       </widget>
       <widget class="QPushButton" name="botonConectar">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <pointsize>15</pointsize>
          <weight>7</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">botonConectar{
font:87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(20,200,220);
border-radius:5px;
border:1px solid white;
}
QpushButton{
font:87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(20,200,220);
border-radius:5px;
border:1px solid white;
}</string>
        </property>
        <property name="text">
         <string>CONECTAR</string>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonBaseDatos">
        <property name="geometry">
         <rect>
          <x>162</x>
          <y>160</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ver historial de limpiezas&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="imagenes_pantalla_principal.qrc">
          <normaloff>:/fotos/fotos_pantalla_principal/estadistica.png</normaloff>:/fotos/fotos_pantalla_principal/estadistica.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ImagenReloj">
        <property name="geometry">
         <rect>
          <x>47</x>
          <y>100</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255);
border-radius:25.4px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="imagenes_pantalla_principal.qrc">:/fotos/fotos_pantalla_principal/reloj.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ImagenConectar">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="imagenes_pantalla_principal.qrc">:/fotos/fotos_pantalla_principal/descarga.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonInfo">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>220</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="imagenes_pantalla_principal.qrc">
          <normaloff>:/fotos/fotos_pantalla_principal/informacion.png</normaloff>:/fotos/fotos_pantalla_principal/informacion.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfig">
   <property name="text">
    <string>Configuracion</string>
   </property>
  </action>
  <action name="actionAcerca_De">
   <property name="text">
    <string>Acerca De</string>
   </property>
  </action>
  <action name="actionRuta_Base_de_Datos">
   <property name="text">
    <string>Ruta Base de Datos</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>botonConfig</tabstop>
 </tabstops>
 <resources>
  <include location="imagenes_pantalla_principal.qrc"/>
 </resources>
 <connections/>
</ui>
